### ISSUE TITLE
Create "Registration form for general practitioners"

### ISSUE THREAD

**Issue by JohanvanBussel** [Sciensano-Healthdata/HD-Corona#3]

**Comment by JohanvanBussel** [comment]
Seperate version for Dutch and French:
- "Registratie formulier voor huisartsen" (NL)
- "Formulaire d'inscription pour les médecins généralistes"

Will only be complete once! 

Final version, but Dutch and French in same form => https://docs.google.com/forms/d/e/1FAIpQLSeEttnhroI3LtOO9CxnCI-EmbNNWsQsoo_uvCSOqkKn-VQwAw/viewform

**Comment by neuronick** [comment]
- Are there any preferences / requirements regarding field names?
  Current field (technical) names for this form:
  - tx_email
  - cd_type_gp
  - tx_gp_name
  - tx_gp_postal_code
  - nr_num_gp_providers
<br>
- Are there any preferences / requirements regarding code values for checkboxes / radios? Currently  numbers starting from 1 are used.
<br>
- Should this form be assessable to anybody (anonymous users)? Is there any form of authentication / authorization needed (user account, secret code, "secret url", eID, something else)? 
<br>
- Does 'Will only be complete once!' mean that we need to enforce this? If yes, what are the criteria? (once per email addres / per IP addres, something else?)

**Comment by JohanvanBussel** [comment]
- OK for field (technical) names
- OK for numbers starting from 1
- form must be assessable to anybody (anonymous users) => No form of authentication / authorization needed (user account, secret code, "secret url", eID, something else)?
- No enforcement of "complete only once"

**Comment by neuronick** [comment]
Here is the first draft of the form:
https://hd4patient-acc.healthdata.be/nl/covid19/general_practitioners/register

**Comment by JohanvanBussel** [comment]
IMPORTANT: "Registration form for general practitioners" should be merged into => Create "Daily status form for general practitioners"
